src/components/inline-link/types.ts

```diff
@@ -1,4 +1,5 @@
 import { TextProps } from 'components/text'
+import React from 'react'
```

Comment:
🚧 Can this be updated to have the `react` import first and also to only import `ReactNode`?

```suggestion
import { ReactNode } from 'react'
import { TextProps } from 'components/text'
```